src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -182,10 +207,19 @@ async def infer_python_dependencies_via_imports(
         ):
             unowned_imports.add(imp)
 
-    unowned_dependency_behavior = python_infer_subsystem.unowned_dependency_behavior
+    return inferred_deps, unowned_imports
+
+
+def _maybe_warn_unowned(
+    address: Address,
+    file: str,
```

Comment:
That's a builtin. Maybe `file_path`?

Reply:
Not anymore... https://docs.python.org/3/library/functions.html#built-in-funcs

```
joshuacannon@CEPHANDRIUS:~$ python
Python 3.8.12 (default, Sep 10 2021, 00:16:05) 
[GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> file
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'file' is not defined
```